gateway-server/src/test/java/org/apache/knox/gateway/session/control/ConcurrentSessionVerifierTest.java

```diff
@@ -48,25 +52,25 @@ private GatewayConfig mockConfig(Set<String> privilegedUsers, Set<String> nonPri
 
 
   @Test
-  public void userIsInNeitherOfTheGroups() {
+  public void userIsInNeitherOfTheGroups() throws ServiceLifecycleException {
     GatewayConfig config = mockConfig(new HashSet<>(Arrays.asList("admin")), new HashSet<>(Arrays.asList("tom", "guest")), 3, 2);
-    verifier.init(config);
+    verifier.init(config, options);
     for (int i = 0; i < 4; i++) {
```

Comment:
The idea was to prove that if the user is in neither of the groups, neither of the limits apply to the user. To test that i verify the user one more time than the higher limit. In a later commit i took out the for cycle because we give the function a token as well, and the tokens are class variables and pre issued in the setUp() function. I'll add some comments to clarify the test and rename it "testuserIsInNeitherOfTheGroupsCanBeLoggedInUnlimitedTimes".